src/pages/mpl-hybrid/update-escrow.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Updating the Configuration of an MPL Hybrid 404 Escrow
+metaTitle: Updating the Configuration of an MPL Hybrid 404 Escrow | MPL-Hybrid
+description: Learn to update the configuration of an MPL 404 Hybrid Escrow account.
+---
+
+The escrow configuration is upgradable via the `updateEscrowV1` plugin.
+
+To make things easier you can fetch your escrow account using the `fetchEscrowV1()` from the `mpl-hybrid` package and use the spread operator to supply all current field values to the update instruction and adjust only the values you wish to change as the original unchanged values will be handled by the spread operator.
+
+## Update Escrow
+
+```ts
+const escrowConfigurationAddress = publicKey("11111111111111111111111111111111");
+
+// Fetch the escrow configuration account.
+const escrowConfigurationData = await fetchEscrowV1(umi, escrowConfigurationAddress);
+
+// Use the spread operator `...` to spread the `escrowConfigurationData` fields into the object
+// and adjust any fields you wish to update.
+const res = await updateEscrowV1(umi, {
+    ...escrowConfigurationData,
+    escrow: hybridEscrowAddress,
```

Comment:
should we mention wherei t is from? Similar to the escrowConfigurationAddress?

Reply:
good spot. This was me renaming and forgetting to rename some other stuff.